src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const trList = Array.from(document.querySelectorAll('tr th')).slice(0, 4);
+const tBody = document.querySelector('tbody');
+const rows = Array.from(tBody.querySelectorAll('tr'));
+
+function parseSalary(salaryString) {
+  return Number(salaryString.replace(/[$,]/g, ''));
+}
+
+trList.forEach((headerTh, columnIndex) => {
+  headerTh.addEventListener('click', () => {
+    // eslint-disable-next-line
+    console.log(columnIndex);
+
+    const sortedRows = rows.sort((a, b) => {
+      const aText = a.children[columnIndex].textContent.trim();
+      const bText = b.children[columnIndex].textContent.trim();
+
+      const aNum = parseSalary(aText);
+      const bNum = parseSalary(bText);
```

Comment:
The `parseSalary` function is used here, which assumes that the text content is a salary string. Ensure that this assumption holds true for all columns, or add a condition to handle non-salary columns differently.